Zend/tests/type_declarations/typed_properties_033.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+Test typed properties yield reference guard
+--FILE--
+<?php
+$foo = new class {
+	public int $foo = 1;
+	public int $bar = 3; 
+	public int $baz = 5;
+	public int $qux = PHP_INT_MAX;
+
+	public function &fetch() {
+		yield $this->foo;
+		yield $this->bar;
+		yield $this->baz;
+		yield $this->qux;
+	}
+};
+
+try {
+	foreach ($foo->fetch() as &$prop) {
+		$prop++;
+	}
+} catch (Error $e) { echo $e->getMessage(), "\n"; }
+
+var_dump($foo);
+?>
+--EXPECTF--
```

Comment:
No need for `EXPECTF`

Reply:
This one is needed as `%d` is used (might not have been there when this comment was written).